dapper/cmd/dapper-api/assets/api-docs/endpoint/data.html

```diff
@@ -0,0 +1,600 @@
+              {
+                "timestamp": 1604135290,
+                "value": "-57"
+              },
+              {
+                "timestamp": 1604135749,
+                "value": "-56"
+              },
+              {
+                "timestamp": 1604137619,
+                "value": "-57"
+              },
+              {
+                "timestamp": 1604141372,
+                "value": "-55"
+              },
+              {
+                "timestamp": 1604142567,
+                "value": "-67"
+              },
+              {
+                "timestamp": 1604146609,
+                "value": "-57"
+              },
+              {
+                "timestamp": 1604148908,
+                "value": "-72"
+              },
+              {
+                "timestamp": 1604154705,
+                "value": "-54"
+              },
+              {
+                "timestamp": 1604155652,
+                "value": "-54"
+              },
+              {
+                "timestamp": 1604158358,
+                "value": "-69"
+              },
+              {
+                "timestamp": 1604159555,
+                "value": "-65"
+              },
+              {
+                "timestamp": 1604161974,
+                "value": "-66"
+              },
+              {
+                "timestamp": 1604165591,
+                "value": "-57"
+              },
+              {
+                "timestamp": 1604167643,
+                "value": "-55"
+              },
+              {
+                "timestamp": 1604168194,
+                "value": "-58"
+              },
+              {
+                "timestamp": 1604170111,
+                "value": "-59"
+              },
+              {
+                "timestamp": 1604173876,
+                "value": "-57"
+              },
+              {
+                "timestamp": 1604174651,
+                "value": "-58"
+              },
+              {
+                "timestamp": 1604175342,
+                "value": "-57"
+              },
+              {
+                "timestamp": 1604176308,
+                "value": "-58"
+              },
+              {
+                "timestamp": 1604177190,
+                "value": "-59"
+              },
+              {
+                "timestamp": 1604178046,
+                "value": "-62"
+              },
+              {
+                "timestamp": 1604179875,
+                "value": "-57"
+              },
+              {
+                "timestamp": 1604181124,
+                "value": "-58"
+              },
+              {
+                "timestamp": 1604182120,
+                "value": "-58"
+              },
+              {
+                "timestamp": 1604186491,
+                "value": "-60"
+              }
+            ]
+          }
+        ]
+      }            
+</pre>
+        </div>
+    </div>
+
+    <div id="footer" class="footer">
+        <div class="row">
```

Comment:
is it possible to put header/footer in a separate template?

Reply:
It's only 3 files and we'll rarely change it so I think I'll leave it as is at this moment.